src/starters.ts

```diff
@@ -54,6 +54,13 @@ export const STARTERS: ReadonlyArray<Starter> = [
     docs: 'https://github.com/stencil-community/stencil-app-starter',
     isCommunity: true,
   },
+  {
+    name: 'ionic',
```

Comment:
to give this a little more information (and not confuse folks about whether or not this is a community vs ionic led project), can we rename this here to 'ionic-pwa'?
```suggestion
    name: 'ionic-pwa',
```